p-token/test-properties/run-proofs.sh

```diff
@@ -59,7 +59,7 @@ fi
 
 set -u
 
-mkdir -p proof_status
+mkdir -p "${PROOF_STATUS_DIR:-proof_status}"
```

Comment:
While we are at it: I think this `proof_status` should be inside the `artefacts` directory (so it can be archived easily), and possibly also use the hashes just like the proof directory.